dependencies.py

```diff
@@ -0,0 +1,48 @@
+import logging
+import sys
+import subprocess
+import json
+
+# Set default logging level to INFO
+logging.basicConfig(level="INFO")
+
+# Check if current python version is 3.8
+if sys.version_info < (3, 8):
+    logging.fatal('McPy needs Python version 3.8.0 or higher to run! Current version is %s.%s.%s' % (
+        sys.version_info.major, sys.version_info.minor, sys.version_info.micro))
+    sys.exit(-4)
+
+# Make sure pip is installed
+logging.info("Making sure pip is installed...")
+try: # Debian and Ubuntu require pip to be installed differently because they disable ensurepip
+    import distro
+    linuxDistro = distro.linux_distribution()
+    if linuxDistro[0] == "Debian" or linuxDistro[0] == "Ubuntu":
+        try:
+            subprocess.check_call(['apt', 'install', 'python3-pip', '-y'])
+        except subprocess.CalledProcessError:
+            logging.fatal("You need to be a sudoer to install pip and the dependencies")
```

Comment:
This needs to be changed into either
> "You need to be the superuser to install pip and the dependencies"

or
> "You need to be root to install pip and the dependencies"

(this will fail on debian if not run under the superuser/root)